noruba/features/lib/step_definitions/command_line.rb

```diff
@@ -85,6 +89,10 @@ def make_arg_list(args)
 class RubyCommand < CommandLine
   def execute(file)
     capture_stdout { require file }
+  rescue RuntimeError => err
+    # no-op: this is raissed when Cucumber fails
+  rescue SystemExit => err
+    # No-op: well, we are ssupposed to exit the rake task
```

Comment:
```suggestion
    # No-op: well, we are supposed to exit the rake task
```

Reply:
Thanks, for the fixes :)

I'm stuck in lockdown with a keyboard on which the "s" key is half broken (it either does not work or does 2 "s" :D). At least the code breaks when this kind of thing happens, it's harder for the comments :D

(I'll merge your changes tomorrow morning, apparently I can't from the mobile :/ )

Reply:
Ouch!

Thanks for your work on this, it's a big job.